src/app/register-email-form/register-email-form.component.ts

```diff
@@ -45,30 +83,127 @@ export class RegisterEmailFormComponent implements OnInit {
         ],
       })
     });
+    this.configService.findByPropertyName('registration.verification.enabled').pipe(
```

Comment:
This setting is currently not shared by the backend, so this will always be "false".  It's unclear to me though if this setting should be moved to the frontend?

Reply:
This is currently a backend setting but the `rest.properties.exposed` property was missing from `rest.cfg` (maybe I added to my local installation only). I fixed the `rest.cfg`and now it should work.